bucket/yara.json

```diff
@@ -35,5 +35,15 @@
     },
     "checkver": {
         "github": "https://github.com/VirusTotal/yara"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/VirusTotal/yara/releases/download/v$version/yara-v$version-win64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/VirusTotal/yara/releases/download/v$version/yara-v$version-win32.zip"
+            }
+        }
```

Comment:
The `-940-` in the current links is an anomaly. Links typically follow this pattern: 

https://github.com/VirusTotal/yara/releases